1-starter-app/app.js

```diff
@@ -0,0 +1,126 @@
+'use strict'
+
+var express = require('express');
+var logger = require('morgan');
+var bodyParser = require('body-parser');
+var cors = require('cors');
+var path = require('path');
+
+var tableClient = require('./service/tableClient')
+var sampleData = require("./configure/data")
+
+var app = express();
+
+// log requests
+app.use(logger('dev'));
+
+app.use(express.static(path.join(__dirname, 'templates')));
+
+app.use('/static', express.static(path.join(__dirname, 'templates')));
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(cors());
+
+
+var listOfKeys=["Temperature","Humidity","Barometer","WindDirection","WindSpeed","Precipitation"];
+
+app.get('/getAllRows',async function(req,res){
+    try{
+        tableClient.getData()
+    .then(res1=>{
+        if(res1.code===0){
+            res.send({code:res1.code,listOfKeys:listOfKeys,entitiesList:res1.data})
+        }else{
+            res.send({code:res1.code,msg:res1.error})
+        }
+    })
+    }catch(err){
+      console.log(err)
+    }
+    
+})
+app.post('/removeEntity',function(req,res){
+    tableClient.removeEntity(req.body.partitionKey,req.body.rowKey).then(()=>{
+        res.send({code:0})
```

Comment:
it seems a bit strange to send `code: 0` as a successful response. `200` or `204` are usually used for success, also `code` is not standard for HTTP reponses. You can use instead

```javascript
res.status(204);
```

Here's a reference to the status codes and their uses: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses

Reply:
Please update this in all the functions in the file